src/commands/config/create.ts

```diff
@@ -0,0 +1,191 @@
+      documentStoreTemporaryDnsParams.address = documentStoreAddress;
+      documentStoreTemporaryDnsParams.publicKey = "";
+      info(`Creating DNS-TXT record..`);
+      verifiableDocumentDnsTxtName = (await createTemporaryDns(documentStoreTemporaryDnsParams)) || "";
+    };
+
+    // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+    const verifiableDnsDid = async () => {
+      documentStoreTemporaryDnsParams.address = "";
+      documentStoreTemporaryDnsParams.publicKey = `did:ethr:0x${walletObject.address}#controller`;
+      info(`Creating DNS-DID record..`);
+      verifiableDocumentDnsDidName = (await createTemporaryDns(documentStoreTemporaryDnsParams)) || "";
+    };
+
+    // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+    const transferableDnsTxt = async () => {
+      info(`Enter password to continue deployment of Token Registry`);
+      const deployTokenRegistryParams = {
+        registryName: "Demo Token Registry",
+        registrySymbol: "DTR",
+        encryptedWalletPath: walletFilePath,
+        network: "ropsten",
+        gasPriceScale: 1,
+        dryRun: false,
+      };
+      const tokenRegistry = await deployTokenRegistry(deployTokenRegistryParams);
+      tokenRegistryAddress = tokenRegistry.contractAddress;
+      success(`Token registry deployed, address: ${highlight(tokenRegistryAddress)}`);
+      info(`Creating temporary DNS for transferable records`);
+      const tokenRegistryTemporaryDnsParams = {
+        networkId: 3,
+        address: tokenRegistryAddress,
+        sandboxEndpoint: "https://sandbox.openattestation.com",
+      };
+      tokenRegistryDnsName = (await createTemporaryDns(tokenRegistryTemporaryDnsParams)) || "";
+    };
+
+    const updatedForms = [] as Form[];
+    for (const form of formsInTemplate) {
+      let updatedIssuers = [] as any[];
+      const updatedForm = form;
+      if (form.type === "VERIFIABLE_DOCUMENT") {
+        updatedIssuers = form.defaults.issuers.map(async (issuer: any) => {
+          if (issuer.identityProof.type === "DNS-TXT") {
+            if (!verifiableDocumentDnsTxtName) await verifiableDnsTxt();
+            issuer.name = "DEMO STORE";
+            issuer.documentStore = documentStoreAddress;
+            issuer.identityProof.location = verifiableDocumentDnsTxtName;
+          } else if (issuer.identityProof.type === "DNS-DID") {
+            if (!verifiableDocumentDnsDidName) await verifiableDnsDid();
+            issuer.name = "Demo Issuer";
+            issuer.id = `did:ethr:0x${walletObject.address}`;
+            issuer.revocation.type = "NONE";
+            issuer.identityProof.location = verifiableDocumentDnsDidName;
+            issuer.identityProof.key = `did:ethr:0x${walletObject.address}#controller`;
+          }
+          return issuer;
+        });
+      }
+      if (form.type === "TRANSFERABLE_RECORD") {
+        updatedIssuers = form.defaults.issuers.map(async (issuer: any) => {
```

Comment:
there is a problem here, if your config file has one document store and one transferable record (the case of TT) then this line will get rid of the promise L 154.